subprojects/file-collections/src/main/java/org/gradle/api/internal/file/collections/FilteredMinimalFileTree.java

```diff
@@ -74,7 +74,7 @@ public VisitType prepareForVisit(FileCollectionInternal.Source source) {
 
             @Override
             public void visitCollection(FileCollectionInternal.Source source, Iterable<File> contents) {
```

Comment:
Wouldn't it make sense to introduce a `MinimalFileTreeStructureVisitor` here that doesn't even have this method?

Reply:
Good idea, I introduced this interface.